adlfs/multithread.py

```diff
@@ -22,8 +22,6 @@
 from .transfer import ADLTransferClient
 from .utils import commonprefix, datadir, read_block, tokenize
 
-MAXRETRIES = 5
```

Comment:
I haven't looked at core, but just to confirm, it will also do something similar with logging when it retries, right? Something about indicating the failure and exception, and that it is trying again?